content/tv/springone-tour/0017.md

```diff
@@ -13,15 +13,68 @@ aliases: ["/tv/springone-tour/17","/s1tour-emea-booternetes-II-the-yaml-strkes-b
 contact: springone@vmware.com
 contact_text: the SpringOne Tour Team
 speakers:
+  - Michael Coté
+  - Olga Maciaszek-Sharma
+  - Marcin Grzejszczak
+  - Jakub Pilimon
+  - Jonatan Ivanov
+  - Nenad Momcilovic 
+  - Timo Salm
+  - Ivan Tarin
 guests: []
-Date: "2021-06-15T10:00:00-07:00"
+Date: "2021-06-15T00:00:00-04:15"
```

Comment:
I think this should be `2021-06-15T00:00:00-07:00` the last part is the TZ so this would be 12am PDT (which I think is 9am CEST). If we want to specify that it runs 4:15 minutes we can set `minutes: 255`